.gitignore

```diff
@@ -7,6 +7,7 @@ coverage
 .idea
 dist
 compiled
+src/generated
```

Comment:
pourquoi l'enlever?

Reply:
Est-ce que la doc va continuer d'avoir /generated?

Reply:
> Pourquoi l'enlever?
Car, par exemple, lorsque l'on fait une PR, ca créer beaucoup de bruit pour rien dans le SCM. Ainsi je pense que c'est mieux que ce soit de le garder under the hood.
> Est-ce que la doc va continuer d'avoir /generated?
Oui, tu peux checker ici: http://www.louisbompart.ovh/sfdx-js/ ce que ca donnera.

Reply:
Btw, tu peux aussi checker https://www.npmjs.com/package/sfdx-js-fake qui me servait de labrat

Reply:
Dac, no worries alors.